src/services/adoption/api.ts

```diff
@@ -0,0 +1,62 @@
+// User can add animal to adoption interest
```

Comment:
@labm1997 A little bit of nitpicking on my part, but should this "Todo list" really be here? I'm guilty of publishing my TODO lists from time to time with my code, so I understand how this could have happened.

Reply:
I missed this todo list. Solved.